mimic/rest/maas_api.py

```diff
@@ -533,28 +533,46 @@ def create_alarm(self, request, tenant_id, entity_id):
         return ''
 
     @app.route('/v1.0/<string:tenant_id>/entities/<string:entity_id>/alarms/<string:alarm_id>',
+               methods=['GET'])
+    def get_alarm(self, request, tenant_id, entity_id, alarm_id):
+        """
+        Gets an alarm by ID.
```

Comment:
Minor nitpick: docstrings should be phrased in the imperative rather than the declarative mood; partially because "implicit you" (make me a sandwich) in a command is more idiomatic than "implicit it" (makes me a sandwich).  However, probably don't change this right now because declarative seems to be the convention within this file and local conventions always win over general style – until they change, anyway :).